File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatMaterial = []

    def __init_(self,license,driver,typeCarAccepted,seatMaterial):
        super.__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatMaterial = seatMaterial

File: Java/UberBlack.java
import java.util.Map;

class UberBlack extends Car {
    // Marca > Modelo y año
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatMaterial;

    public UberBlack(String license, Account driver,
                    Map<String, Map<String,Integer>> typeCarAccepted,
                    ArrayList<String> seatMaterial){
        super(license,driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    }
}